contract_manager/src/chains.ts

```diff
@@ -231,16 +211,25 @@ export class SuiChain extends Chain {
       BigInt(fee),
       BigInt(exponent)
     ).encode();
-    return this.wrapWithWormholeGovernancePayload(3, setFee);
+    return setFee
   }
 
   generateGovernanceSetDataSources(datasources: DataSource[]): Buffer {
     const setDataSource = new SetDataSources(
       this.wormholeChainName,
       datasources
     ).encode();
-    return this.wrapWithWormholeGovernancePayload(2, setDataSource);
+    return setDataSource
   }
+
+  generateGovernanceSetStalePriceThreshold(newValidStalePriceThreshold: bigint): Buffer {
```

Comment:
You can also move this to the parent Chain class, so that it can be used on other chains too